GitHubA11yFixes.user.js

```diff
@@ -157,6 +114,85 @@ function onNodeAdded(target) {
 	}
 }
 
+function onNodeAdded(target) {
+	var elem;
+	if(document.location.pathname.match(/\/(?:[^\/]+\/[^\/]+\/)?notifications\/?/)){
+		// Notifications page.
+		//First, make sure the h3 with the repo becomes an h2, so we can set the notifications for that repos as an h3.
+		for(elem of target.querySelectorAll(".notifications-repo-link"))
+			makeHeading(elem.parentNode, 2);
+		//Now, set the link marking each notification as an h3.
+		for(elem of target.querySelectorAll(".list-group-item-link")){
+			// Make the parent span a heading, but aria-owns the image to the end of this heading for niceity.
+			makeElementOwn(elem.parentNode, [elem, elem.previousElementSibling]);
+			makeHeading(elem.parentNode, 3);
+		}
+		//firefox   overrides title as if it was aria-label. Remove title here because it might mask the issue contents or other info.
+		//This unfortunately breaks visual content, but it doesn't really matter.
+		elem.removeAttribute("title"); 

```

Comment:
1. What is `elem` supposed to be here? It's outside the previous loop, but that loop didn't restrict the scope of `elem`, so I think this will affect the last item in that loop.
2. Regarding title, are you referring to the Firefox issue where title gets set as the label of headings? That has since been fixed; e.g. `<h2 title="foo">bar</h2>` results in "bar" in browse mode. So, this code can possibly be removed.